src/runtime/storageNG/drivers/driver-factory.ts

```diff
@@ -37,9 +37,16 @@ export abstract class Driver<Data> {
     this.storageKey = storageKey;
     this.exists = exists;
   }
-  abstract registerReceiver(receiver: ReceiveMethod<Data>): void;
+  abstract registerReceiver(receiver: ReceiveMethod<Data>, token?: string): void;
   abstract async send(model: Data, version: number): Promise<boolean>;
 
+  // Return a token that represents the current state of the data.
+  // This can be provided to registerReceiver, and will impact what
```

Comment:
Does this mean that registering multiple receivers will mean each subsequent call can clobber the token from the previous call?

Reply:
Oh, nevermind. Looking at volatile - it seems like only one receiver is allowed to be registered at a time.  Maybe this should be `setReceiver`?

Reply:
Yeah, I think you're right.